undefined_syms.txt

```diff
@@ -795,6 +795,9 @@ D_06003128 = 0x06003128;
 D_060042AC = 0x060042AC;
 D_060059B0 = 0x060059B0;
 
+// z_en_goroiwa
+D_0400D340 = 0x0400D340;
+
```

Comment:
`z_bg_jya_goroiwa` already defines another symbol used by `z_en_goroiwa`: `D_060006B0 = 0x060006B0;`

I don't know what the policy on this is, should the symbol be defined twice if that's possible?

Reply:
The way it was explained to me was that you only added a symbol if it wasn't elsewhere but somebody correct me if I'm wrong.

Reply:
Since both actors use the same object, D_060006B0 is the same symbol so there is no need to define it twice.